jenkins/ci.opensuse.org/openstack-devstack.yaml

```diff
@@ -42,6 +42,6 @@
               du $file
               exit 1
           fi
-
+          exit $ret
```

Comment:
I think, it could get stuck in line 31 if one of the services started by qa_devstack.sh does not properly daemonize (detach from the console / process group)